docker-compose-local-db.sample.yml

```diff
@@ -30,7 +30,7 @@ services:
       MEMCACHED_PORT: 11211
       PORT: 8080
       NEXT_API_BASE_URL: gateway.docker.internal:3000/api
-      NEXT_REVALIDATE_SECRET_TOKEN: super-secret-token
+      NEXT_SECRET_TOKEN: super-secret-token
```

Comment:
Renamed NEXT_REVALIDATE_SECRET_TOKEN to NEXT_SECRET_TOKEN because it is not used in both the revalidation and reviewing context